modules/AOS_PDF_Templates/generatePdf.php

```diff
@@ -266,7 +266,7 @@ function populate_group_lines($text, $lineItemsGroups, $lineItems, $element = 't
     return $text;
 }
 
-function populate_product_lines($text, $lineItems, $element = 'tr')
+function populate_product_lines($text, $lineItems, $element = 'tr', $template)
```

Comment:
Hi @jack7anderson7,

We are adding this argument here, but we don't change the place that calls this method.